tests/FamilyHubs.ReferralApi.UnitTests/WhenUsingReferralCommands.cs

```diff
@@ -80,9 +79,9 @@ public async Task ThenGetReferralsByReferrer()
             var testReferral = GetReferralDto();
             CreateReferralCommand command = new(testReferral);
             CreateReferralCommandHandler handler = new(mockApplicationDbContext, mapper, logger.Object);
-            var id = await handler.Handle(command, new System.Threading.CancellationToken());
+            await handler.Handle(command, new System.Threading.CancellationToken());
 
-            GetReferralsByReferrerCommand getcommand = new("Bob Referrer",1,10);
+            GetReferralsByReferrerCommand getcommand = new("Bob Referrer",1,10, default!, default!);
```

Comment:
default! = this value is null but dont complain about it being null, is that the intention? if so you can close this comment, if the default value should be empty string or default datatime then set it so

please stop using default! it breaks my brain :)

Reply:
Replaced these with null